peps/pep-0744.rst

```diff
@@ -133,12 +133,20 @@ Like the rest of the interpreter, the JIT compiler is generated at build time,
 and has no runtime dependencies. It supports a wide range of platforms (see the
 `Support`_ section below), and has comparatively low maintenance burden. In all,
 the current implementation is made up of about 900 lines of build-time Python
-code and 500 lines of runtime C code.
+code and 500 lines of runtime C code. The current approach is optimized for 
```

Comment:
I debated putting this here or in Rejected Ideas? Rejected Ideas didn't seem like quite the right place. Happy to shift this around if you think there's a better spot.

Reply:
I think it's fine here if it's just one sentence. If you add a multi-paragraph discussion of AOT compilation, "Rejected ideas" would be a better fit.

Reply:
Eh, I sorta think it would be better somewhere else, personally. It feels a bit weird to essentially say "we're just doing it this different way" in a section titled "Rationale".

I'd either put it in a (short!) Rejected Ideas section or work it into the last paragraph of the Motivation section (where this question is already strongly hinted at).

Also, from a technical perspective, the problem isn't really that the compiler is "optimized" for JIT compilation. The problem is that supporting AOT would be a massive ground-up redesign of the existing specialization and micro-op tracing infrastructure that we already have.